core/src/wallet/ripple/explorers/api/RippleLikeBlockParser.h

```diff
@@ -49,14 +49,13 @@ namespace ledger {
                     // Ledger index is not really a hash but since XRP doesn't have reorg
                     // it's safe to use ledger index as a unique hash.
                     _block->hash = number;
-                }
-                return true;
-            }
-
-            bool String(const rapidjson::Reader::Ch *str, rapidjson::SizeType length, bool copy) {
-                std::string value = std::string(str, length);
-                if (getLastKey() == "close_time_human") {
-                    _block->time = DateUtils::fromJSON(value);
+                } else if (getLastKey() == "close_time") {
+                    std::string number(str, length);
+                    BigInt value = BigInt::fromString(number);
+                    // rippled is using another epoch time than UNIX
+                    // https://xrpl.org/basic-data-types.html#specifying-time
+                    std::chrono::system_clock::time_point time(std::chrono::seconds(value.toUint64() + 946684800));
```

Comment:
Do that magic number is used somewhere else ? In that case we probably should store it within `constexpr` variable.